CONTRIBUTING.md

```diff
@@ -30,9 +38,50 @@ The following are required to build the devfile index container image containing
     - This will also validate the devfiles in this repository, making sure they conform to a minimum standard.
     - This step will also be run in the PR build for the repository.
 
-5) (Optional) Push the container image to a container registry and [deploy](https://github.com/devfile/registry-support#deploy) a custom devfile registry on to Kubernetes or OpenShift. Test odo with the devfile stack hosted in that registry.
-  
-    - For now, strictly an optional requirement, but recommended.
-    - PR tests will also verify this.
+5) Open a pull request against this repository with a brief description of the change.
+
+
+### Updating
+
+Updating an existing devfile stack is relatively straightforward:
+
+1) Find the stack under the `stacks/` folder that you wish to update
+2) Make the necessary changes to the stack (such as updating image tags, commands, starter projects, etc)
+3) Update the version of stack, following the [semantic versioning format](https://semver.org/)
+4) Test your changes:
+    
+    - Minimally, testing with odo (`odo create`, `odo push`, etc) is recommended, however if your Devfile is used with other tools, it's recommended to test there as well
```

Comment:
I read `minimally` a couple times. I think it'd be simpler to say,
`Testing with odo, like `odo create` and `odo push`, is recommended. However, if you use your devfile with other tools, you can test it with those tools.`

Reply:
You `should` test it, rather than `can` test it?

Reply:
If it's recommended for users to test with the other tools, I then yes, say `should` :)